src/arch/xtensa/configs/rembrandt_defconfig

```diff
@@ -11,7 +11,7 @@ CONFIG_FORMAT_CONVERT_HIFI3=n
 CONFIG_LP_SRAM=n
```

Comment:
update for latest sof FW is something very generic and wont be very useful as a commit message in few months.

Please add something more specific. 

e.g drivers: amd: Fix AMD drivers

... give here reason for the fix pointing to commits that caused the change.